sky/skylet/job_lib.py

```diff
@@ -230,11 +230,11 @@ def get_statuses_json(job_ids: List[Optional[int]]) -> str:
     statuses = {job_id: None for job_id in job_ids}
     for (job_id, status) in rows:
         statuses[job_id] = status
-    return json.dumps(statuses)
+    return common_utils.encode_payload(statuses)
 
 
 def load_statuses_json(statuses_json: str) -> Dict[int, JobStatus]:
```

Comment:
nit: Since we no longer return a valid json from these methods, I'm wondering if we should rename all these functions from `<name>_json` to `<name>_payload` (e.g. `get_statuses_payload`). wdyt?

Reply:
Good point! Renamed